### PR TITLE
Add tooltip to ldap server wizard

### PR DESCRIPTION
**Pull request by Henni**
fixes regression #19684
I didn't test this as I haven't set up ldap in my test environment.

@PVince81 do we want a backport for this?

### CODE REVIEW

**Comment by PVince81** [comment]
I don't think this is the correct approach.
In the past the tooltip was set on a broad range of elements, globally, possibly one of these https://github.com/owncloud/core/pull/18122/files#diff-c493b80ad1a9bcfa6b69637097972371L1281
Here you are setting it only on a single field "ldap_host", but the other fields will still be missing the tooltip.

One idea is to bring back the "global" approach, but restrict it to the LDAP wizard container.

**Comment by Henni** [comment]
@PVince81 what do you think of this solution?

**Comment by PVince81** [comment]
Getting closer, but not the best. The problem here is that the LDAP wizard code will also affect input elements outside the wizard. Please scope the selector to the LDAP wizard container only.

**Comment by Henni** [comment]
I was able to partly test this.
@PVince81 I think this should finally fix #19684

**Comment by PVince81** [comment]
Any reason why you are using `:input` instead of simply `input` ? (`:input` doesn't benefit from performance boost, see https://api.jquery.com/input-selector/)

**Comment by PVince81** [comment]
Ok I see, it's for all form elements, input, select, etc. It's fine.

Let me have a quick test.

**Comment by PVince81** [comment]
For some reason the tooltip stays when the field is focused instead of being just on-hover.
This can produce effects like this:
ldap-tooltip-overlap

I checked OC 8.1 and back then it was only on hover, not sticky.
Maybe the call is missing an option to make it behave like that ?

I see you added a container value, this fixes https://github.com/owncloud/core/issues/18044

**Comment by Henni** [comment]
@PVince81 here you go. Now the tooltip should only trigger on hover.

**Comment by PVince81** [comment]
:+1: works

@Henni can you squash these two commits ? The overall change is only a single line, so it would be cleaner :smile:

**Comment by Henni** [comment]
@PVince81 sure... just a second

**Comment by PVince81** [comment]
Needs a second reviewer @blizzz

File: apps/user_ldap/js/wizard/view.js [hunk 1]
```diff
@@ -352,6 +352,7 @@ OCA = OCA || {};
 			$('.ldap_submit').button();
 			$('.ldap_action_test_connection').button();
 			$('#ldapSettings').tabs({ beforeActivate: this.onTabChange });
+			$('#ldapWizard1 :input').tooltip({placement: "right", container: "body", trigger: "hover"});
```
Comment (blizzz) [inline apps/user_ldap/js/wizard/view.js]: not enough, the tabs in the wizard are numbered, and this only the first one is covered. Plus there are the "older" advanced and expert tabs which are also missed this way. Replace it with #ldapSettings and we are good.

**Comment by Henni** [comment]
@blizzz could you take another look?

**Comment by blizzz** [comment]
@Henni looks good, thanks! :+1:

**Comment by blizzz** [comment]
@karlitschek backport to 8.2?

**Comment by karlitschek** [comment]
yes. please backport :+1:

**Comment by blizzz** [comment]
@Henni would you do this?

**Comment by Henni** [comment]
@blizzz I'm on it

**Pull request merged**